server/data/visitSchedulerApiClient.test.ts

```diff
@@ -450,6 +451,28 @@ describe('visitSchedulerApiClient', () => {
     })
   })
 
+  describe('updateLocationGroup', () => {
+    it('should update location group', async () => {
+      const updateLocationGroupDto = TestData.updateLocationGroupDto()
+      const singleLocationGroup = TestData.locationGroup()
+
+      fakeVisitSchedulerApi
+        .post(`/admin/location-groups/group/${singleLocationGroup.reference}`, <UpdateLocationGroupDto>{
+          name: updateLocationGroupDto.name,
+          locations: updateLocationGroupDto.locations,
+        })
+        .matchHeader('authorization', `Bearer ${token}`)
+        .reply(201, singleLocationGroup)
```

Comment:
```suggestion
        .reply(200, singleLocationGroup)
```
Doesn't really matter in this context, but I think updating returns an `HTTP 200`